raiden/console.py

```diff
@@ -228,8 +227,8 @@ def ping(self, peer):
             print("Error: peer {} not found in discovery".format(peer))
             return
 
-        nonce = self._ping_nonces[peer]
-        self._ping_nonces[peer] += 1
+        nonce = self._raiden.protocol.ping_nonces[peer]
```

Comment:
I think your PR could be a good moment to lift `ping(peer)` into the main API and move it out of the console tools.

Otherwise I'd suggest to encapsulate getting & +1-ing the ping nonces in a function, to avoid manual messing with the nonces in potentially multiple places i.e. 
```
def next_ping_nonce(self, peer):
  """Get ping nonce for peer and increase the internal counter.
  """
  …
```